index.html

```diff
@@ -10,300 +11,294 @@
+        </section>
+        <!-- contact form -->
+        <section class="contact-me">
+            <div class="background-image">
+                <div class="container">
+                    <div class="scontainer">
+                        <h2 class="heading-2">Contact me</h2>
+                        <p class="wordings">
+                            If you have an application you are interested in developing, a feature
+                            that you need built or a project that needs coding. I’d love to help
+                            with it
+                        </p>
+                    </div>
+                    <form action="https://formspree.io/f/mgedjkyb" id="form1" method="post" class="contact-form">
+                        <div class="input-control">
+                            <input type="text" id="name" placeholder="Your Name" name="name" required />
                         </div>
-                        <ul class="language-list">
-                            <li class="content">
-                                <img class="icon" src="Images/Ellipse-1.png" alt="JavaSript-logo">
-                                <p class="text">JavaScript</p>
-                            </li>
-                            <li class="content">
-                                <img class="icon" src="images/HTML-logo.png" alt="html-logo">
-                                <p class="text">HTML</p>
-                            </li>
-                            <li class="content">
-                                <img class="icon" src="Images/CSS-logo.png" alt="css-logo">
-                                <p class="text">CSS</p>
-                            </li>
-                        </ul>
-                    </li>
-                    <li>
-                        <div class="frame-7">
-                            <h4 class="languages">Frameworks</h4>
-                            <a class="arrow-down" href="#"><img src="Images/arrow-right.png" alt="Arrow Right Icon">
-                            </a>
+                        <div class="input-control">
+                            <input type="email" id="email" placeholder="Your Email" name="email" required />
                         </div>
-                    </li>
-                    <li>
-                        <div class="frame-7">
-                            <h4 class="languages">Skills</h4>
-                            <a class="arrow-down" href="#"><img src="Images/arrow-right.png" alt="Arrow Right Icon"></a>
+                        <div class="input-control">
+                            <textarea placeholder="Write your message here" id="message" name="message" maxlength="500"
+                                required></textarea>
                         </div>
-                    </li>
-                </ul>
+                        <div class="input-control">
+                            <input type="submit" value="Get It Touch" />
+                        </div>
```

Comment:
- [ ] Kindly ensure your form button is center justified like in the Figma template, currently it is to the left
![image](https://user-images.githubusercontent.com/90556221/178115813-b62bc2e5-babb-4070-a50c-3ef75c6eacee.png)